testbook/client.py

```diff
@@ -104,13 +105,15 @@ def execute_cell(self, cell, **kwargs) -> Union[Dict, List[Dict]]:
         for idx in cell_indexes:
             try:
                 cell = super().execute_cell(self.nb['cells'][idx], idx, **kwargs)
-            except CellExecutionError as e:
+            except CellExecutionError as ce:
                 # Look for in-built Python exception
+                nbexception = None
```

Comment:
Default to either `ce` or `RuntimeError(ce.value)` instead of None imo so you always have a raw exception with the message available.

Reply:
Yeah makes sense, I have defaulted it to `CellExecutionError`.